frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -11,7 +11,8 @@ import {
   legendColorScale,
   riskSorting,
   mapMetrics,
-  isWaitingOrDisabled as _isWaitingOrDisabled,
+  isWaiting as _isWaiting,
+  isDisabled as _isDisabled,
```

Comment:
This is a nit, but we try to avoid having a leading or trailing `_` in names. I'd suggest changing the var to `waiting` below and leaving the helper `isWaiting` if this is due to naming conflicts.